sass/homepage/_testimonial.scss

```diff
@@ -0,0 +1,61 @@
+.testimonial {
+  background-color: $color-background-light;
+  padding: 10rem 4rem;
+
+  &-content {
+    display: grid;
+    grid-template-columns: repeat(3, min-content);
+    justify-content: center;
+    gap: 2rem;
+    margin: 5rem auto;
+  }
+
+  &__card {
+    display: flex;
+    flex-direction: column;
+    justify-content: end;
+    gap: 2rem;
+    width: 28rem;
+    height: 38rem;
+    background-color: #fff;
+    padding: 2rem;
+    border-radius: 10px;
+    box-shadow: 0 1rem 1rem rgba(0, 0, 0, 0.158);
+    position: relative;
+
+    &::before {
+      content: "\201C";
+      position: absolute;
+      top: -9rem;
+      left: -0.5rem;
+      font-size: 24rem;
+      color: rgba($color-background-light, 0.19%);
+    }
+  }
+
+  &__details {
+    display: grid;
+    grid-template-columns: repeat(2, max-content);
+    grid-template-rows: repeat(2, min-content);
+    column-gap: 1rem;
+
+    &--name {
+      grid-row: 1/2;
+      grid-column: 2/3;
+      align-self: end;
+    }
+
+    &--star {
+      align-self: start;
+      justify-content: start;
+    }
+
+    &--photo {
+      grid-row: 1/-1;
+      grid-column: 1/2;
+      width: 8rem;
+      height: 8rem;
+      border-radius: 150px;
+    }
```

Comment:
Could you organize this classes in order  we visualize them in the grid, --photo, --name and start.

Reply:
Yes, I did not noticed it was in this order, sorry for the silly mistake

Reply:
Not a problem this is not a mistake it is just a better way to organize the code.